application/dash_application/views/preqc_wgs.py

```diff
@@ -448,13 +515,18 @@ def generate_ploidy(df, colour_by, shape_by):
                         EMPTY_WGS, initial_colour_col, initial_shape_col)
                 ),
             ]),
-
-            # DataTable for all samples info
-            html.Div(className="data-table",
-                children=[
+            html.Div(className='terminal-output',
+                     children=[
+                         html.Pre(generate_terminal_output(EMPTY_WGS, initial_cutoff_rpsp, initial_cutoff_insert_mean,
+                                                           initial_cutoff_pf_reads),
+                                  id=ids['terminal-output'],
+                                  )
+                     ]),
+            html.Div(className='data-table',
+                     children=[
                     build_table(ids["data-table"], wgs_table_columns, WGS_DF,
                                 BAMQC_COL.TotalReads)
-                ])
+                     ]),
```

Comment:
There seems to be a `])` pair missing or added somewhere, as the terminal-output and datatable live outside the `div className="seven columns"`  block

Reply:
Fixed.